app/models/like.rb

```diff
@@ -0,0 +1,5 @@
+class Like < ApplicationRecord
```

Comment:
- [ ] You forgot to set up the associations between `user and like` you should do that for example the like  can belong to the user and the post  so it should be the code like this :
```RUBY
`belongs_to :author, class_name: 'User'`
```